pydis_site/apps/api/viewsets/bot/off_topic_channel_name.py

```diff
@@ -69,11 +68,11 @@ def get_object(self) -> OffTopicChannelName:
         name = self.kwargs[self.lookup_field]
         return get_object_or_404(queryset, name=name)
 
-    def get_queryset(self) -> QuerySet:
+    def get_queryset(self, **kwargs) -> QuerySet:
         """Returns a queryset that covers the entire OffTopicChannelName table."""
-        return OffTopicChannelName.objects.all()
+        return OffTopicChannelName.objects.filter(**kwargs)
 
-    def create(self, request: HttpRequest) -> Response:
+    def create(self, request: Request, *args, **kwargs) -> Response:
```

Comment:
The `*args` and `**kwargs` you add here, where do they come from? This is called from REST framework.

Reply:
(same question for `list`)

Reply:
They come from the function definition within the framework.

Reply:
Sane and good.